data/json/vehicle_parts.json

```diff
@@ -4880,6 +4880,25 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "chimes",
+        "name" : "chimes",
+        "symbol" : "&",
+        "color" : "red",
+        "broken_symbol" : "&",
+        "broken_color" : "light_gray",
+        "damage_modifier" : 0,
+        "durability" : 40,
+        "epower" : -200,
+        "item" : "chimes",
+        "difficulty" : 4,
+        "flags" : ["CHIMES"],
+        "breaks_into" : [
+            {"item": "steel_lump", "min": 3, "max": 5},
+            {"item": "steel_chunk", "min": 3, "max": 5},
+            {"item": "scrap", "min": 3, "max": 5}
```

Comment:
I agree with both of you.

The update of stereos can be moved out of this PR if you want, so it's definitely not a blocker for current PR.